pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -154,10 +154,19 @@ def export_testcase(
     """
 
     os.makedirs(out_dir, exist_ok=True)
+    input_names = kwargs.pop(
+        'input_names',
+        ['input_{}'.format(i) for i in range(len(args))])
 
     onnx_graph, outs = _export(
         model, args, strip_large_tensor_data, large_tensor_threshold,
-        **kwargs)
+        input_names=input_names, **kwargs)
+
+    # Remove unused inputs
+    used_input_index_list = [
+        input_names.index(input.name) for input in onnx_graph.graph.input]
+    input_names = [input_names[i] for i in used_input_index_list]
+    args = [args[i] for i in used_input_index_list]
```

Comment:
is it better to check `assert len(args) == len(input_names)` before?

Reply:
I added here https://github.com/pfnet/pytorch-pfn-extras/pull/86/files#diff-4b250f25742298aa0ef7ea68986407898b68b8d35420d0384804d1e29d4f3671R160.